src/main/java/net/minecraftforge/fml/common/network/handshake/NetworkDispatcher.java

```diff
@@ -122,6 +123,7 @@ public NetworkDispatcher(NetworkManager manager)
         this.manager = manager;
         this.scm = null;
         this.side = Side.CLIENT;
+        this.modList = new HashMap<>();
```

Comment:
You can use `Collections.emptyMap()` for a non-null initial value, rather than making empty `HashMap`s